### ISSUE TITLE
How to let the device screen show fast under remote web page?

### ISSUE THREAD

**Issue by boxerxing** [openstf/stf#377]
Hi developer,

 i found a problem as title,  I hope it works like the touch event when i'm using from remote web page to  control a device. 

Descriptions:

1. I start stf with command :  stf local --public-ip <my ip>
2. I login stf from another computer with web (Actually, the login process is very slow.)
3. select one device and then try to control it by touch screen, the action is worked as fast by it's screen show sync need to long time with drop frame.

Could you give me some advance?

**Comment by sorccu** [comment]
Yeah, STF hasn't been optimized for that scenario. It was originally made for internal networks only.

**Comment by boxerxing** [comment]
@sorccu  Thanks this feedback ,  maybe this is some mistake in my question, both my compute are in the same network.

**Comment by boxerxing** [comment]
my ip like "10.252.162.10" and another ip is "10.252.163.5"

**Comment by vbanthia** [comment]
This problem can also occur in internal networks if your wifi is too weak or it is being shared by many people. STF client side is not optimised for weak network connection.

**Comment by codeskyblue** [comment]
reduce the image size which produced by the stf client is also a way. @boxerxing

**Comment by FengJianjian** [comment]
I get the same problem. But I can't find the way to reduce the image size @codeskyblue.

**Comment by codeskyblue** [comment]
You need to change the code. @FengJianjian

**Comment by yuyangcq** [comment]
我使用命令：stf provider --name localhost.localdomain --min-port 7400 --max-port 7700 --connect-sub tcp：//172.50.1.159：7114 --connect-push tcp：//172.50.1.159 ：7116 --group-timeout 900 --public-ip 172.50.1.159 --storage-url http://172.50.1.159:7100/ --adb-host 172.17.244.11 --adb-port 5037 --vnc-initial尺寸600x800-静音大师-不允许远程

然后就一直停在“ tcp：//172.50.1.159：7114”接收输入，过了几分钟就报错了，这个是什么原因
![2021-05-26_104729](https://user-images.githubusercontent.com/29696080/119603939-c3509080-be20-11eb-82f6-a04f41900c47.png)

**Comment by xiaohuanxiong3** [comment]
I get the same problem.
